readme.txt

```diff
@@ -37,10 +37,13 @@ This is compatible with WooCommerce>=2.4, including the new 3.0 release. It has
 
 == Upgrade Notice ==
 = 2.0.0 =
-* Switches from WooCommerce side currency conversion to Razorpay's native multi currency support. 
+* Switches from WooCommerce side currency conversion to Razorpay's native multi currency support.
 
 == Changelog ==
 
+= 2.4.1 =
+* Tested upto WordPress 5.4 and WooCommerce 3.7.1
```

Comment:
Just change to ,   Updated WordPress support version info

Reply:
Updated.